blocks/columns/columns.css

```diff
@@ -2,6 +2,10 @@
   margin: 2rem 0 4rem;
 }
 
+.columns.smaller-bottom-margin {
```

Comment:
Is the `smaller-bottom-margin` something you see repeating across pages or is it a one-off pattern?

Reply:
TBH, I only saw this page as a possible candidate for this 
https://www.sunstar.com/about/chairman-interview/. (the image and text below the quote)
But yeah maybe 2 cases doesn't warrant a whole another style; but the bottom spacing is way too big as is for my use-case on the history page


![Screenshot 2023-10-11 at 10 15 11](https://github.com/hlxsites/sunstar/assets/44490413/0d913182-3064-483d-85e4-bd072dae5d90)

Reply:
That big spacing doesn't look fine in any case. Could you check who added that spacing and we can see if they are ok removing/minimizing it?

Reply:
Seems it was added here for the engineering website

https://github.com/hlxsites/sunstar/commit/282eb539f675209dec44790f86c4679152efa1e9#diff-95c2da244c71fe94382a46e48c77292b821711db947818095f7b3842fc42fc44R75

Reply:
And here
https://github.com/hlxsites/sunstar/commit/119f126edd8b28f26462c3c546638a041c9fc276#diff-95c2da244c71fe94382a46e48c77292b821711db947818095f7b3842fc42fc44R2

I'd also change this to `margin: 2rem 0 2rem` after removing the 7.5 rem one as well